docker-compose.yaml

```diff
@@ -296,8 +296,9 @@ services:
       POSTGRES_DB: ${CSV_POSTGRES_DB}
     volumes:
       - /home/user/postgres-data:/var/lib/postgresql/postgres-data-csv
+      - /home/user/postgres-backup:/var/lib/postgresql/data
```

Comment:
Double check the correct way to provide path volumes, i suggest reading [here](https://docs.docker.com/engine/storage/volumes/).